profile/E-ARK-eHealth1-REPRESENTATION.xml

```diff
@@ -350,6 +351,7 @@
             </requirement>         
             <requirement ID="EH17" REQLEVEL="MUST" RELATEDMAT="VocabularyContentInformationTypeSpecification" EXAMPLES="fileSecExample1">
                 <description>
+                <!--Example does not contain a fileGrp with the @USE attribute value 'Representations'-->

```

Comment:
The representation mets file doesn't have a representations file group, this is used in the root mets to point to the representation mets files. There is a problem here as the CSIP does not discriminate between root and representation mets requirements and so when validating a rep mets it would fail due to lack of a file group with @USE='Representations'. This needs to be raised as an issue in CSIP.